global.h

```diff
@@ -179,7 +179,11 @@ EXTERN int g_nproc_t, g_nproc_x, g_nproc_y, g_nproc_z;
 EXTERN int g_sourceid, g_sourceid2, g_sourceid_step, Nsave;
 EXTERN int g_spintrace;
 EXTERN int g_currentgammas;
-EXTERN int g_currentgammabasis;
+typedef enum cvc_GammaBasis {
```

Comment:
can you place the enum definition into ```include/enums.hpp``` ? Of course, that will then have to be included at the top of this file, just below ```types.h```.